js-runtime/index.js

```diff
@@ -456,6 +459,39 @@ module.exports = {
   hashf,
   hashv,
 
+  // In Node.js these opcodes don't have to be IO opcodes, but in the Rust runtime they do, because
```

Comment:
non-blocking nit: In Node.js these opcodes -> In Node.js the datastore opcodes

Reply:
Will fix